filecoin/retriever.go

```diff
@@ -287,6 +291,8 @@ func (retriever *Retriever) retrieveFromBestCandidate(ctx context.Context, retri
 				err,
 			)
 			stats.Record(ctx, metrics.RetrievalDealFailCount.M(1))
```

Comment:
what does this metric become? Just curious. Isn't it essentially already FailedRetrievalsPerRequestCount?